package.json

```diff
@@ -92,6 +92,7 @@
     "@babel/preset-env": "^7.11.0",
     "@babel/preset-react": "^7.0.0",
     "@babel/register": "^7.10.5",
+    "@octokit/core": "^3.1.2",
```

Comment:
accidental add?

Reply:
Nope.  It's a peer dependency of something from danger :-/